internal/app/parcel_carrier/service.go

```diff
@@ -0,0 +1,24 @@
+package parcel_carrier
+
+import (
+	"context"
+	"parcel-service/internal/app/model"
+	svc "parcel-service/internal/app/service"
+	"time"
+)
+type service struct {
+	repo svc.CarrierParcelAcceptRepository
```

Comment:
I think CarrierAcceptRepository is good enough to understand. No need to mention parcel for each name. Or it may like, UserAcceptsCarrierRepository.

Reply:
Renamed it